Framework/DataHandling/src/SaveDiffCal.cpp

```diff
@@ -187,7 +124,7 @@ void SaveDiffCal::writeIntFieldFromTable(H5::Group &group,
   std::vector<int32_t> data;
   column->numeric_fill(data);
   data.erase(data.begin() + m_numValues, data.end());
```

Comment:
What do you think about passing `m_numValues` to numeric_fill and not calling erase?